.env.example

```
@@ -83,9 +83,22 @@ GOOGLE_GEMINI_API_KEY=""
+# model name — glm-*, qwen-*, gpt-* alike — is sent to that gateway
+# (`useAnthropicSdk` in packages/core/src/llm.ts). Two consequences:
+#   1. A model the gateway does not serve fails there, not at DashScope/OpenAI.
+#      Measured on the Z.ai Coding Plan endpoint 2026-08-17: `qwen3-coder`
+#      returns HTTP 400, so the code-level default is unreachable on that
+#      deployment — always set KEA_MODEL explicitly when using a gateway.
+#   2. Gateways alias silently rather than 404. On that same endpoint
+#      glm-5.1/glm-5.2/glm-5 were all served by glm-5.3, and claude-haiku-4-5
+#      by glm-4.7 — with a 200 and no warning. Trust the `served` field in the
+#      llm subsystem's warning log, not the name you set here; the cost ledger
+#      prices what you REQUESTED.
+KEA_MODEL="glm-4.5-air"              # glm-4.5 | glm-4.5-air | glm-4.5-flash | glm-4.7 | qwen3-coder | claude-haiku-4-5
```

dotenv-linter (4.0.0)

[warning] [ValueWithoutQuotes] This value needs to be surrounded in quotes

(ValueWithoutQuotes)